packages/manager/modules/pci/src/components/project/instance/instance.constants.js

```diff
@@ -5,6 +5,40 @@ export const CAPABILITIES = {
   FAIL_OVER_IP: 'failoverip',
 };
 
+/* TODO: Applied those status instead of hardcoded */
+export const INSTANCE_STATUS = {
```

Comment:
Do we need the list of all the status for this purpose ? Also I don't understand the todo

Reply:
The goals, is to provide a list of instance status to avoid to compare instance.status with hardcoded value.
like ```this.instance.status === 'ACTIVE'``` instead we will use ```this.instance.status === INSTANCE_STATUS.ACTIVE```

Reply:
i got your point of view @marie-j.
I let only the status that i implement for this feature.
The rest should be updated into a dedicated Jira task (created).

Thanks.

Reply:
Resolved.